conda/core/subdir_data.py

```diff
@@ -549,15 +571,15 @@ def _process_raw_repodata(self, repodata, state=None):
                     )
                     continue
 
-                package_kwargs = dict(info.items())
-                # Python doesn't like duplicate keyword arguments? (PackageRecord(**kwargs, fn=x))
-                package_kwargs.update({"fn": fn, "url": join_url(channel_url, fn)})
-                package_record = PackageRecord(**package_kwargs)
-
-                _package_records.append(package_record)
-                _names_index[package_record.name].append(package_record)
-                for ftr_name in package_record.track_features:
-                    _track_features_index[ftr_name].append(package_record)
+                # lazy
+                # package_record = PackageRecord(**info)
+                info["fn"] = fn
```

Comment:
should consider copying this

Reply:
Hm, no idea if we have any constraints on copying, but if there were, that promise is very likely broken somewhere and then probably not worth keeping.

Reply:
Will keep it as is. I'm more confident that we don't need to copy.